vignettes/paper.md

```diff
@@ -83,6 +89,8 @@ Currently, only single-layer clustering is supported (e.g., for `method = "mle"`
 With a Bayesian approach (`method = "bayes"`), `abn` utilizes its own implementation of the Laplace approximation as well as the `INLA` package [@inla2013] to fit a single-level hierarchical model for Binomial, Poisson, and Gaussian distributed variables. 
 Independent of the type of data, multinomial distributed variables are not yet implemented with `method ="bayes"` (details in the [online manual](https://r-bayesian-networks.org/quick_start_example.html)).
 
+[//]: # (Line 90 needed?)
```

Comment:
We can remove it in favor of the word count. It is mentioned in more detail on the homepage anyway.